homeassistant/components/upnp/translations/en.json

```diff
@@ -1,26 +1,25 @@
 {
     "config": {
+        "flow_title": "{name} ({host})",
         "abort": {
             "already_configured": "UPnP/IGD is already configured",
-            "incomplete_device": "Ignoring incomplete UPnP device",
             "no_devices_discovered": "No UPnP/IGDs discovered",
-            "no_devices_found": "No UPnP/IGD devices found on the network.",
-            "no_sensors_or_port_mapping": "Enable at least sensors or port mapping",
-            "single_instance_allowed": "Only a single configuration of UPnP/IGD is necessary."
+            "no_devices_found": "No UPnP/IGD devices found on the network."
         },
         "step": {
-            "confirm": {
-                "description": "Do you want to set up UPnP/IGD?",
-                "title": "UPnP/IGD"
-            },
             "init": {
                 "title": "UPnP/IGD"
             },
+            "ssdp_confirm": {
+                "data": {
+                    "name": "Name"
+                },
+                "title": "UPnP/IGD",
+                "description": "Do you want to set up this UPnP/IGD device?"
+            },
             "user": {
                 "data": {
-                    "enable_port_mapping": "Enable port mapping for Home Assistant",
-                    "enable_sensors": "Add traffic sensors",
-                    "igd": "UPnP/IGD"
+                    "unique_id": "Host"
```

Comment:
This translation is wrong?

Reply:
The unique_id (USN) is stored, but the Host is presented.

Reply:
It was right. I'd chosen Host as that is the most recognizable. Showing a USN, such as `uuid:upnp-InternetGatewayDevice-1_0-889ffacb9043::urn:schemas-upnp-org:device:InternetGatewayDevice:1` would be very user friendly.

I've changed it to the name of the device. It requires an initialization of the device (getting XML etc), but it is only once.